data/transactions/logic/README.md

```diff
@@ -191,6 +191,7 @@ bytes on outputs.
 | `b==` | A is equals to B, where A and B are byte-arrays interpreted as big-endian unsigned integers => { 0 or 1} |
 | `b!=` | A is not equal to B, where A and B are byte-arrays interpreted as big-endian unsigned integers => { 0 or 1} |
 | `b%` | A modulo B, where A and B are byte-arrays interpreted as big-endian unsigned integers. Fail if B is zero. |
+| `base64_decode v` | the base64 decoding of X with the alphabet denoted by V (imm arg). Fail if X is not base64 encoded with alphabet V |
```

Comment:
I'll change the `a` to `v` but keep the esoteric capitalization conventions.

Reply:
Actually, I can't use `A` because that refers to a stack variable according to the [table's preamble](https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/README.md#arithmetic-logic-and-cryptographic-operations) so it could be confusing to the reader. I'm proposing `e` instead and changing _alphabet_ to _encoding alphabet_